### PR TITLE
Correct import xml event lifecycle

### PR DESCRIPTION
**Pull request by MaxTru**
I spotted this missing testcoverage and inconsistency in the event lifecycle and wanted to clean it up prior to finishing the dmn-js promisification. This allows me to have a stable baseline for the promisification exercise.

### CODE REVIEW

File: packages/dmn-js-shared/src/base/Manager.js [hunk 1]
```diff
@@ -109,15 +109,20 @@ export default class Manager {
...
+        this._emit('import.done', { error: err, warnings: parseWarnings });
 
         return done(err, parseWarnings);
       }
 
       var view = this._activeView || this._getInitialView(this._views);
 
       if (!view) {
-        return done(new Error('no displayable contents'));
+        var noDisplayableContentsErr = new Error('no displayable contents');
```
Comment (nikku) [inline packages/dmn-js-shared/src/base/Manager.js:120]: Great catch.

I believe I found a similar inconsistency in bpmn-js as we introduced our modern import :smile:.

As `var` in ES5 gets hoisted to the outer scope my proposal would be to simply override the existing `err` variable. We should avoid declaring a nested `var`.

**Review by nikku — CHANGES_REQUESTED** [review]
Great stuff and good improvement on our existing test coverage :clap:.

It is not tested, it is likely broken :skull:

File: packages/dmn-js-shared/src/base/Manager.js (hunk 1 above)
Comment (MaxTru) [reply packages/dmn-js-shared/src/base/Manager.js]: Thanks, good point. I corrected that.

**Review by nikku — APPROVED** [review]
:clap:

**Pull request merged**